.vsts-ci-templates/build_and_test.yml

```diff
@@ -33,3 +33,20 @@ steps:
     ls *_test | xargs -n1 -I '{}' sh -c 'printf "\n*** Binary: {} ***\n "; llvm-cov-7 report -object {} -instr-profile {}.profdata -ignore-filename-regex="(boost|openenclave|3rdparty|/test/)"'
   displayName: Coverage
   workingDirectory: build
+
+- script: |
+    ls | grep profraw | xargs -n1 > prof_files
```

Comment:
I think this is probably large enough to deserve its own script

Reply:
better?